scripts/werf-in-image/dockerfile/werf/centos.Dockerfile

```diff
@@ -19,10 +19,17 @@ RUN setcap cap_setuid+ep /usr/bin/newuidmap && \
     setcap cap_setgid+ep /usr/bin/newgidmap && \
     chmod u-s,g-s /usr/bin/newuidmap /usr/bin/newgidmap
 
-RUN useradd build
+RUN useradd -m build && \
+    groupadd -g 1001 github-runner && \
+    useradd -u 1001 -g 1001 -m github-runner && \
+    mkdir -p /home/build/.local/share/containers /home/build/.werf && \
+    mkdir -p /home/github-runner/.local/share/containers /home/github-runner/.werf && \
+    chown -R build:build /home/build && \
+    chown -R github-runner:github-runner /home/github-runner
```

Comment:
Does GitHub needs a user named `github-runner` with home dir `/home/github-runner`? Can't we have something more generic like user `build1001`, group `build1001`, homedir `/home/build1001`? If we can, let's do this.